### ISSUE TITLE
`pretext publish` should just `git push`

### ISSUE THREAD

**Issue by oscarlevin** [PreTeXtBook/pretext-cli#132]
I have a project with a branch set to push to a different repository on github for publishing.  I've set the git push.default to correctly push my branch to the correct remote on the correct branch, but then `git push origin` throws an error.  I think if we let users set up their git to work (or assume they haven't changed anything) then just `git push` would be sufficient.

**Comment by StevenClontz** [comment]
Yeah, line https://github.com/PreTeXtBook/pretext-cli/blob/main/pretext/project.py#L251 basically emulates using the Git CLI, so I should be able to change line https://github.com/PreTeXtBook/pretext-cli/blob/main/pretext/project.py#L273 similarly to do whatever `git push` wants to do.
